### DISCUSSION TITLE
Loing Post requests

### DISCUSSION THREAD

**Discussion by defaul0t** [projectdiscovery/katana#449] [General]
I found that the crawler is not ideal for capturing the request triggered by the click function, such as the post request sent by the important login button, if the window with the verification code

test_url : http://sctgf.tcl.com:8088/#/login?from=%2Flogin

It's a pity that it didn't get the post request

But another similar tool can achieve

https://github.com/chaitin/rad

**Comment by MetzinAround** [comment]
Converting to a discussion since this doesn't directly connect to a change or PR. 

@defaul0t, is there a question or desired action to be taken based on this?

**Reply by defaul0t** [reply]
Hi, do you have time to test this feature, I think the post request will help uncover more issues

**Comment by ehsandeep** [comment]
@defaul0t This is known and currently, katana make post request in fuzzy way and something to improve in future for headless mode, there is ticket to add a feature that will later help to make post request more accurately - https://github.com/projectdiscovery/katana/issues/13

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
